docs/_blog/spending_policy_demo.md

```diff
@@ -31,11 +31,11 @@ In a real-world wallet a longer relative time-lock would probably be used, but w
 ### Step 0: Install a recent version `bdk-cli`
 
 ```bash
-cargo install bdk-cli --features repl,electrum,esplora
+cargo install bdk-cli --features esplora
```

Comment:
I did not get why `esplora` is required here.. Electrum is enough for the example it seems??

Reply:
agreed should be fine to use `--features electrum` .. but if you do want to use esplora then it needs to be `--features esplora-ureq` or `--features esplora-reqwest`

Reply:
Ok so then I think I should remove the features completely right? Alekos' comment was that now the electrum feature is a default so we should remove it.